drone/interface.go

```diff
@@ -246,7 +246,7 @@ type Client interface {
 	TemplateList(namespace string) ([]*Template, error)
 
 	// TemplateCreate creates a template.
-	TemplateCreate(template *Template) (*Template, error)
```

Comment:
do we need name in here, similar to templateupdate ?

Reply:
The name is passed in on the POST as part of the payload. 
The template exists on update so it can be part of the URL.